src/handler.rs

```diff
@@ -8,49 +8,49 @@ use arbiter::Arbiter;
 use context::Context;
 use fut::{self, ActorFuture};
 
-/// Message handler
+/// A trait for objects which can handle messages of a specific type.
```

Comment:
object is not common term for Rust

I'd suggest to keep it simple and rephrase it as:

A trait that describes how to handle message

Reply:
I think my description is ok, see the description of `std::io::Write` for example:

> A trait for objects which are byte-oriented sinks.

I think an "object" in Rust just means an instance of any type.